src/status_im/subs.cljs

```diff
@@ -89,6 +89,7 @@
 (reg-root-key-sub :waku/v2-peer-stats :peer-stats)
 (reg-root-key-sub :visibility-status-updates :visibility-status-updates)
 (reg-root-key-sub :navigation2/navigation2-stacks :navigation2/navigation2-stacks)
+(reg-root-key-sub :ui/old? :ui/old?)
```

Comment:
@Parveshdhull don't we already have such subscription ?

Reply:
Yes, `status-im.utils.config/new-ui-enabled?`

Reply:
Shortsighted of me to not find such subscription. I should definitely replace the `:ui/old?` with `new-ui-enabled?`

Reply:
Hey folks, I replaced the unnecessary state to use `config/new-ui-enabled?` and it all works.

I noticed `config/new-ui-enabled?` is not available as a subscription from re-frame's db. Instead, it's a Clojure atom, which shouldn't be used in subscriptions, but it is (e.g. `:home-items`). Instead, a Reagent atom is preferable, because then re-frame can keep track of derefs automatically and re-render more efficiently. You probably know about this, but I just wanted to let you know just in case.

I also found something that might be a bug for another PR, the `:home-items` subscription is not dereferencing `config/new-ui-enabled?`, which means the check always returns true. Should this be reported in an issue? See https://github.com/status-im/status-mobile/blob/develop/src/status_im/subs.cljs#L1579

Reply:
> I also found something that might be a bug for another PR, the :home-items subscription is not dereferencing config/new-ui-enabled?, which means the check always returns true. Should this be reported in an issue? See https://github.com/status-im/status-mobile/blob/develop/src/status_im/subs.cljs#L1579

@Parveshdhull, @flexsurfer, FYI

Reply:
`(and config/new-ui-enabled? (= view-id :chat-stack)))`

Hi, Thank you very much for finding this issue, yes `config/new-ui-enabled?` should be dereferenced here.
But interestingly, it didn't affect the resulting output, because even when `config/new-ui-enabled?` always remains true, for net output to be true `(= view-id :chat-stack)` also needs to be true. And view-id is only `chat-stack` in the new UI. 
Still, thank you for finding this, feel free to create a separate PR or it will be fixed once this [commit](https://github.com/status-im/status-mobile/pull/13753/files#diff-21ed1701d49fc16290b71c7c74bfae9dc355e07910b0d4507e0d29a83aacc3bf) is merged.
> Should this be reported in an issue?

Yes, the issue sounds good, easy to track the fix.
Thank you

Reply:
Awesome @Parveshdhull, I think I'll leave for that commit to be merged given that you pointed out it's not currently affecting users.